libs/core/src/lib/card/card-footer.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit, ChangeDetectionStrategy, ElementRef } from '@angular/core';
+
+import { applyCssClass, CssClassBuilder } from '../utils/public_api';
+
+import { CLASS_NAME } from './constants';
+
+@Component({
+    selector: 'fd-card-footer',
+    template: '<ng-content></ng-content>',
+    changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
can we use here `host: { class: CLASS_NAME.cardFooter }` instead of CssClassBuilder?